application/views/surveyAdministration/tabCreate_view.php

```diff
@@ -198,6 +198,9 @@
 
         $('#addnewsurvey').on('submit',  function(event){
             event.preventDefault();
+            // Disable both buttons. Normally there's no need to re-enable them. The 'save-form-button' may already be disabled by it's onclick event.
+            $('#create-survey-submit').prop('disabled', true);
+            $('#save-form-button').addClass('disabled').attr('onclick', 'return false;');
```

Comment:
Question : `$('#save-form-button').addClass('disabled').off('click');` didn't work ?

Reply:
Haven't tried it. Not sure it is worth trying.
The button has some JS onthe onclick attribute, so guess would be good to keep on in the same line when handling events.

Reply:
> Haven't tried it. Not sure it is worth trying. The button has some JS onthe onclick attribute, so guess would be good to keep on in the same line when handling events.

`$('#save-form-button').addClass('disabled').off('click').removeAttr('onclick');` ?

Not important …